SECP-service/src/main/java/com/visucius/secp/UseCase/LoginRequestController.java

```diff
@@ -0,0 +1,96 @@
+package com.visucius.secp.UseCase;
+
+import com.visucius.secp.DTO.LoginRequestDTO;
+import com.visucius.secp.DTO.TokenDTO;
+import com.visucius.secp.daos.UserDAO;
+import com.visucius.secp.models.User;
+import com.visucius.secp.util.PasswordUtil;
+import org.apache.commons.lang3.StringUtils;
+import org.jose4j.lang.JoseException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.ws.rs.WebApplicationException;
+import javax.ws.rs.core.Response;
+
+public class LoginRequestController {
+
+    private final static Logger LOG = LoggerFactory.getLogger(LoginRequestController.class);
+    private UserDAO userDAO;
+    private TokenController tokenController;
+
+    //error messages
+    private static final String LOGIN_FAIL_NO_CREDENTIALS = "Login Failed. Please provide your username and password.";
+    private static final String LOGIN_FAIL_NO_USERNAME = "Login Failed. Please provide your username.";
+    private static final String LOGIN_FAIL_NO_PASSWORD = "Login Failed. Please provide your password.";
+    private static final String LOGIN_FAIL_USER_NOT_FOUND = "Login Failed. username does not exist.";
+    private static final String LOGIN_FAIL_WRONG_PASSWORD = "Login Failed. Incorrect password.";
```

Comment:
We can have a resource file with all the constants. This is more of a design opinion but something we can consider.